backend/src/model/dto/plantings.rs

```diff
@@ -46,6 +46,8 @@ pub struct PlantingDto {
     pub additional_name: Option<String>,
     /// Is the planting an area of plants.
     pub is_area: bool,
+    /// Notes about the planting in Markdown.
+    pub notes: Option<String>,
```

Comment:
i think we have to name it plant_note since there should also be a note for seeds in the future.

Reply:
so a planting will have a plant_note and a seed_note

Reply:
I'll ask Yvonne if the seeds_note shouldn't directly edit the notes in seeds.

Reply:
She also said that the comment of the seeds should directly reflect the note of the inventory.
Of course this mapping could also be done in the backend, the frontend simply writing to seed_notes.

Reply:
Ok so what does that mean?

In the database we now have `notes` in `Seeds` and `Plantings`, but only here in `PlantingDto` it should be called `planting_notes` to distinguish it clearly from seed notes? Should I also populate the field `seed_notes`?

Reply:
> Ok so what does that mean?

To fix what @danielsteinkogler wanted should be easy:

```suggestion
    pub planting_note: Option<String>,
```

With this rename, afaics, the PR already implements #1067.

> Should I also populate the field seed_notes?

To also implement the seed_note better in another PR (as it is a bit more tricky to get consistent when several plantings have the same seed_id).

@danielsteinkogler please make sure that the issues for the backend are up-to-date and describe correctly what you want.

Ideally test if this PR does what you want, please approve the PR when you are happy.